lib/readline.js

```diff
@@ -429,7 +428,9 @@ Interface.prototype._tabComplete = function() {
           }
         }
         handleGroup(self, group, width, maxColumns);
+      }
 
+      {
```

Comment:
> let vs var shouldn’t matter 

I don't think so because here the brackets would create a scope with `let` declaration but not for `var`. For example:

```js
{
  var f = completions.filter(function(e) { if (e) return e; });
  var prefix = commonPrefix(f);
}
// here we actually can access `f` and `prefix`, this is not what we expect by brackets.
// using `let` instead of `var` can make it expected result.
```

Actually my suggestion is not removing brackets here and changing `var` -> `let`.